apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx

```diff
@@ -63,6 +64,18 @@ export default class AdminCohortView extends React.Component {
     }
   }
 
+  sanitizeStringForCsv = str => {
+    let res = str;
+    if (str) {
+      // Convert line breaker to dot, multiple spaces to single space
+      res = res.replace(/\n|\r\n|\r+/gm, '.');
```

Comment:
There's a bug here - did you mean `(\n|\r\n|\r)+`? As-is, you'll match multiple consecutive carriage returns `\r` but not multiple consecutive line breaks in other formats.

Reply:
it's a bug, thanks for catching it